light-node/src/config.rs

```diff
@@ -41,7 +41,8 @@ pub struct LightNodeConfig {
 pub struct LightClientConfig {
     /// Address of the Tendermint fullnode to connect to and
     /// fetch LightBlock data from.
-    pub address: tendermint::net::Address,
+    #[serde(with = "tendermint_proto::serializers::from_str")]
```

Comment:
Can we rather have a custom `Serialize/Deserialize` instance for `tendermint_rpc::Url`, so that one does not have to provide such an attribute everywhere we want to store it within a serializable struct?

Reply:
Makes sense, will do! :+1: